### PR TITLE
Add pr_target_branch option

### PR DESCRIPTION
**Pull request by raphink**

### CODE REVIEW

File: lib/modulesync/cli.rb [hunk 1]
```diff
@@ -99,6 +99,8 @@ class Base < Thor
              :type => :array,
              :desc => 'Labels to add to the pull/merge request',
              :default => CLI.defaults[:pr_labels] || []
+      option :pr_target_branch,
```
Comment (bastelfreak) [inline lib/modulesync/cli.rb:102]: can you please also put an example in the README.md ?

**Comment by raphink** [comment]
I don't understand why this is failing... I'll see about it later.

**Comment by raphink** [comment]
There, it should be ready now @bastelfreak !

File: README.md [hunk 2]
```diff
@@ -185,17 +185,38 @@ GitLab automatically with the `--pr` CLI option.
...
+```
+
+Then:
 
 * Set the PR/MR title with `--pr-title` or in `modulesync.yml` with the
   `pr_title` attribute.
 * Assign labels to the PR/MR with `--pr-labels` or in `modulesync.yml` with
   the `pr_labels` attribute. **NOTE:** `pr_labels` should be a list. When
   using the `--pr-labels` CLI option, you should use a comma separated list.
+* Set the target branch with `--pr_target_branch` or in `modulesync.yml` with
```
Comment (ekohl) [inline README.md:217]: Is this really `--pr_target_branch` or `--pr-target-branch`?

Comment (raphink) [reply README.md:217]: It seems equivalent, considering all other options are declared with an underscore as well...

**Pull request merged**